### ISSUE TITLE
Left Menu gone on tablets and 2 more questions...

### ISSUE THREAD

**Issue by MrDalle** [Ysurac/FlightAirMap#286]
Hey there!

1. 
Is there any way to access the left menu while using the map on a smartphone or tablet?
I can access the main menu with statistics etc. but the other one is gone..
 
2.
Is it possible to scale plane icons in 3d mode or color them like you can in settings.php for the 2D mode? 
 
3. Also most of the planes doesnt get displayed like 787-A380 etc.. it shows a prop plane instead.. any idea? im using phpvms as a source.

Thanks in advance!

**Comment by Ysurac** [comment]
1. What browser are you using ? This is working with chrome & firefox.
2. Not yet. This will be possible soon, need to check latest Cesium features.
3. In 2D or 3D ? With phpVMS it's not easy to get plane model. I use *aircraftname* field to get it but It's possible some types are not supported. I you give me aircraft_icao used I can fix this.

**Comment by MrDalle** [comment]
1. Working fine on a Ipad Air with safari and firefox. Iphone its not appearing on both browsers. Same with a samsung Tablet (a6) i believe its not working due the smaller screen size. Forcing the page to Desktop mode works on all devices!

2.Sounds great!

3. Only in 3D. If you enter the "Track it by link" mode most planes are props due they are not recognized..
Maybe its wokring by naming the planes different right in the phpvms database? Is there any "list" on what the correct identifiers are?

**Comment by Ysurac** [comment]
1. I only test on a Samsung Galaxy Tab 3, so a 8". The a6 is 10", should be working.
3. If I had a list I would use it. When you click on a plane with props model, what is the aircraft icao ?

**Comment by MrDalle** [comment]
1. Its working now! maybe it was a bug on my site but i changed globalMap3Ddefault = FALSE;  it was TRUE before.

2. attached an example. The easiest way to fix this could be naming the planes like they are on the ivao example since its wokring there. 

<img width="1446" alt="bildschirmfoto 2017-03-28 um 16 31 16" src="https://cloud.githubusercontent.com/assets/7756672/24410892/83972026-13d4-11e7-9d95-c66c2e5147af.png">

**Comment by Ysurac** [comment]
1. I never tested 3D mode on an android device
2. Strange, this one should work. Some are working ? You really have a file **models/330.glb** ?

**Comment by MrDalle** [comment]
Yes the models are on the server! 737 and A320 is working fine! but all 747-787 A380 are props..

Attached the structure of a phpvms databse.. maybe that helps? 
If i change the registration an image of the aircraft should show up too or?

<img width="1482" alt="bildschirmfoto 2017-03-28 um 17 11 53" src="https://cloud.githubusercontent.com/assets/7756672/24413175/f2a2a048-13da-11e7-8ba1-88765ba0c706.png">

**Comment by Ysurac** [comment]
All seems ok.
Even without changing the registration an image should show if it's enabled.

I will make some test with a phpvms feed.

**Comment by MrDalle** [comment]
Ok sounds good!

**Comment by Ysurac** [comment]
image should now work with latest commit.

**Comment by Ysurac** [comment]
phpVMS **/action.php/acars/data** only give me **"aircraftname":"B777-200"** but not the ICAO, so I try to get the ICAO from this, but I don't have a full list of aircraftname. So if you give me faulty ICAO I should be able to fix them.

**Comment by MrDalle** [comment]
pictures are working! 

For the ICAOS i could send you a dump of our phpvms_aicraft table. We inserted almost all planes.. (like in the picture) would that help?

**Comment by Ysurac** [comment]
Should help yes.

**Comment by MrDalle** [comment]
Ok there you go:
https://we.tl/OHPN1dXLob

Which picture got shown depends on Aircraft Registration right?

**Comment by Ysurac** [comment]
aircraft registration and if not found aircraft icao and airline, if no airline only aircraft icao.
